myExpenses/src/main/java/org/totschnig/myexpenses/fragment/ContextualActionBarFragment.kt

```diff
@@ -31,7 +32,7 @@ abstract class ContextualActionBarFragment : Fragment(), OnGroupClickListener, O
     @JvmField
     var expandableListSelectionType = ExpandableListView.PACKED_POSITION_TYPE_NULL
     private val menuSingleIds = intArrayOf(R.id.EDIT_COMMAND,
-            R.id.CREATE_PLAN_INSTANCE_EDIT_COMMAND,
+            R.id.CREATE_PLAN_INSTANCE_EDIT_COMMAND, R.id.CREATE_PLAN_INSTANCE_SAVE_COMMAND,
```

Comment:
This change seems to have been caused by an outdated upstream. Please remove it.